pkg/init/Dockerfile

```diff
@@ -1,3 +1,10 @@
+FROM linuxkit/alpine:630ee558e4869672fae230c78364e367b8ea67a9 AS build
+# RUN mkdir -p /out/etc/apk && cp -r /etc/apk/* /out/etc/apk/
```

Comment:
remove unused line?  Though it seems like we use this in other images so I'm wondering if we need this here